packages/core/src/external/commonwell/document/document-downloader-local.ts

```diff
@@ -100,22 +100,23 @@ export class DocumentDownloaderLocal extends DocumentDownloader {
     }
 
     const old_extension = path.extname(fileInfo.name);
-    const [detectedFileType, detectedExtension] = detectFileType(downloadedDocument);
+    const documentBuffer = Buffer.from(downloadedDocument);
```

Comment:
just moving Buffer.from from inside detectFileType to outside it right here, so that CW and CQ can share this detectFileType function. Also moved the function to return an object instead of an array